### DISCUSSION TITLE
Can the SDK support users searching for other models?

### DISCUSSION THREAD

**Discussion by SekiroZ** [pixiv/vroid-sdk-developers#97] [Q&A]
hi,I'm using VROID to develop a virtual human social app and have a few questions to ask.

Q1: Because there are some gamification scenarios, do I need to apply for additional multiplayer SDK?

Q2: How can the users of our app use the open model on VRoidhub?

Q3: Does the SDK have a search function so that users can search for model keywords and then replace the image

Q4: How can I synchronize the likelist in the web and the likelist in the SDK. weblike has more than ten likelists, but the SDK has only a few.

First time using VRoid problem has a little more, thanks for guidance.

ths~

**Comment by nyamadan** [comment]
Hi.

> Q1: Because there are some gamification scenarios, do I need to apply for additional multiplayer SDK?

Yes, you need to apply additional multiplayer SDK.

> Q2: How can the users of our app use the open model on VRoidhub?

Your application may not have passed our review yet.
Please see [this document](https://vroid.pixiv.help/hc/en-us/articles/900000213643-VRoid-SDK-Guidelines#:~:text=After%20passing%20this%20reviewing%2C%20the%20application%20is%20published%20on%20VRoid%20Hub%2C%20which%20will%20increase%20its%20exposure%20and%20enable%20the%20use%20of%20special%20models%20such%20as%20the%20following.).

> Q3: Does the SDK have a search function so that users can search for model keywords and then replace the image

VRoid SDK does not provide searching models.

> Q4: How can I synchronize the likelist in the web and the likelist in the SDK. weblike has more than ten likelists, but the SDK has only a few.

I think it is the same as Q2.

**Reply by SekiroZ** [reply]
ths !!

About Q2 I have a few details still want to understand the next

Do I need to integrate the SDK into the app and then give it to the VRoid team for review, if so how should I submit it after I finish the app?

We are an iOS app

**Reply by nyamadan** [reply]
> Do I need to integrate the SDK into the app and then give it to the VRoid team for review, if so how should I submit it after I finish the app?
> We are an iOS app

Yes, We review apps that we download from AppStore.

Please inquire through [the inquiry form](https://www.pixiv.net/support?mode=inquiry&type=43&service=vroid-sdk) for more information.
